pull_request_template.md

```diff
@@ -2,9 +2,9 @@
 
 ## Changes Proposed:
 This PR proposes changes to:
--  [ ] The core (units, players, creatures, game systems).
--  [ ] To scripts (bosses, spell scripts, creature scripts).
--  [ ] To the database (SAI, creatures, etc).
+-  [ ] Core (units, players, creatures, game systems).
+-  [ ] Scripts (bosses, spell scripts, creature scripts).
+-  [ ] Database (SAI, creatures, etc).
```

Comment:
Perhaps a bit ambiguous for creatures to be listed as an example for both the core and database for new users.